pkgs/servers/xmpp/prosody/default.nix

```diff
@@ -27,17 +24,17 @@ let
 in
 
 stdenv.mkDerivation rec {
-  version = "0.9.12";
+  version = "0.10.0";
   name = "prosody-${version}";
 
   src = fetchurl {
     url = "http://prosody.im/downloads/source/${name}.tar.gz";
-    sha256 = "139yxqpinajl32ryrybvilh54ddb1q6s0ajjhlcs4a0rnwia6n8s";
+    sha256 = "1644jy5dk46vahmh6nna36s79k8k668sbi3qamjb4q3c4m3y853l";
   };
 
   communityModules = fetchhg {
     url = "https://hg.prosody.im/prosody-modules";
-    rev = "9a3e51f348fe";
+    rev = "a844d1535c4d";
     sha256 = "09g4vi52rv0r3jzcm0bsgp4ngqq6iapfbxfh0l7qj36qnajp4vm6";
```

Comment:
No checksum update?

Reply:
Strangely not, at the time I updated, `nix-prefetch-hg https://hg.prosody.im/prosody-modules` returned the checksum that was already there. No idea how that could happen, but the build worked. 🤔

Just updated the `communityModules` again when resolving the merge conflicts, now the checksum changed. Maybe the prosody module hosting was broken or something…